tests/TestCase/ORM/Behavior/TranslateBehaviorTest.php

```diff
@@ -827,6 +827,113 @@ public function testFindSingleLocaleBelongsToMany()
         $this->assertEquals('Translated Info', $result->tags[0]->special_tags[0]->extra_info);
     }
 
+    /**
+     * Tests that parent entity isn't dirty when containing a translated association
+     *
+     * @return void
+     */
+    public function testGetAssociationNotDirtyBelongsTo()
```

Comment:
Can't we just add the dirty flag check to the above existing test cases?

Reply:
We could. I'll reduce the amount of duplication in the tests. I think they were initially provided to help show the cases that were and were not working.